lib/bindings/src/breez_liquid_sdk.udl

```diff
@@ -345,6 +363,34 @@ enum PaymentState {
     "Complete",
     "Failed",
     "TimedOut",
+    "Refundable",
+};
+
+dictionary SwapRefundInfo {
+    string id;
```

Comment:
Perhaps the swap btc address instead of id? I think it is more intuitive for the user.

Reply:
IMO the `id` is useful to associate this with the `Payment` from `list_payments`.

So I would keep `id`, maybe for clarity also rename it to `swap_id`, as that's the `Payment` field it would match.

Reply:
@ok300 but we don't have id on Payment. We do have swap_id though.

Reply:
Yes, the suggestion was to add `swap_id` to the `RefundableSwap` / `SwapRefundInfo`, in addition to the `swap_address`.

Without it, people cannot cross-reference the refundable info with the payment info from `list_payments`.